### PR TITLE
Portuguese Translation Update

### PR DESCRIPTION
**Pull request by Ustklm**
Added more translations
Minor fixes of languages.

### CODE REVIEW

**Comment by yairm210** [comment]
You probably missed a " somewhere if the build is failing

**Comment by Ustklm** [comment]
Alright, I'm checking it right now.

File: android/assets/jsons/Translations/Diplomacy,Trade,Nations.json [hunk 1]
```diff
@@ -808,6 +829,7 @@
 		Czech:"Kulturní"
 		German:"Kultiviert"
 		Korean:"문화적"
+
```
Comment (Stalinbr19391945) [inline android/assets/jsons/Translations/Diplomacy,Trade,Nations.json]: Portuguese:"cultivado

**Comment by yairm210** [comment]
We've changed the way the translation files are handled for a future with less conflicts and less failing builds!
Instructions at https://github.com/yairm210/Unciv/wiki/Translating =D

**Pull request closed without merging**